### ISSUE TITLE
RuntimeError: Failed to initialize CinderX

### ISSUE THREAD

**Issue by HShan886** [facebookincubator/cinder#130]
`python -X jit-help`  which is compiled by lastest code (commit id: 3674f93)  will report error 
```
Error in sitecustomize; set PYTHONVERBOSE for traceback:
RuntimeError: Failed to initialize CinderX
```

**Comment by tekknolagi** [comment]
Can repro with `docker run -it --rm ghcr.io/facebookincubator/cinder-runtime:cinder-3.10 -X jit-help` or `docker run -it --rm ghcr.io/facebookincubator/cinder-runtime:cinder-3.10.3674f93 -X jit-help`

**Comment by jbower-fb** [comment]
Thanks for the report. It looks like this issue is very specific to starting with `-X jit-help`, if you aren't using that arg then things should be working (baring some *other* bug).

**Comment by jbower-fb** [comment]
Fixed with bea832f32ca252d5d6e2c7f2b46ce4fe7a339132.
